src/styles/ui.css.js

```diff
@@ -184,6 +188,16 @@ export default css`
   padding: 0.4em 0.7em;
 }
 
+.respec-warning-list > li:last-child,
+.respec-error-list > li:last-child {
+  margin-bottom: 0;
+}
+
+.respec-warning-list > li:first-child,
+.respec-error-list > li:first-child {
+  margin-top: 0;
+}
```

Comment:
How about using [lobotomized owl selector](https://alistapart.com/article/axiomatic-css-and-lobotomized-owls/):
```suggestion
.respec-warning-list > li + li,
.respec-error-list > li + li {
  margin-top: 0.5rem;
}
```

(Will also need following change at line 184/188)
```diff
 .respec-warning-list > li,
 .respec-error-list > li {
   padding: 0.4em 0.7em;
+  margin: 0;
 }
```

Reply:
This looks neat, but I'm not following this? 

The current problem is the padding seen here: 

<img width="174" alt="Screen Shot 2021-12-21 at 5 59 53 pm" src="https://user-images.githubusercontent.com/870154/146885899-a93cd713-b174-4e33-9d77-ab0bdfb2a5fd.png">
 
If the above fixes that, then sure... but li:first-child and li:last-child is more explicit IMO? (i.e., the rest are all fine)

Reply:
Above (my suggestion) fixes the white margin issue.

> li:first-child and li:last-child is more explicit IMO

But longer. I think the lobotomized owl selector is fairly common (it's basically saying: add a `margin-top` on every `li` that is preceded by a `li` - so that excludes the first; above link has good diagrams if you want to see how it works).
_[Even tailwind people use it](https://tailwindcss.com/docs/space)._

Reply:
“lobotomized owl” hehe, never heard of this. I’ll take a look - same with tailwind (never heard of it).